src/openforms/registrations/contrib/email/templatetags/data_summary.py

```diff
@@ -0,0 +1,15 @@
+from django import template
+from django.template.loader import get_template
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def data_summary(context):
```

Comment:
```suggestion
def email_data_summary(context):
```
I'd consider namespacing this templatetag too - it's intended for e-mail, I can imagine there being a `data_summary` for different contexts too :thinking: 

We do already have the `{% summary %}` tag which is used for the confirmation e-mail - maybe that should be renamed/aliased to `confirmation_summary` and this one named `registration_summary`?